cookbook/composing-stores.md

```diff
@@ -43,11 +40,11 @@ const useY = defineStore('y', () => {
 })
 ```
 
-## Nested Stores
+## Almacenes anidados {#nested-stores}
 
-Note that if one store uses another store, you can directly import and call the `useStore()` function within _actions_ and _getters_. Then you can interact with the store just like you would from within a Vue component. See [Shared Getters](#shared-getters) and [Shared Actions](#shared-actions).
+Ten en cuenta que si un almacén utiliza otro almacén, puedes importar directamente y llamar a la función `useStore()` dentro de _acciones_ y _getters_. Entonces puedes interactuar con el almacén como lo harías desde dentro de un componente Vue. Ver [Getters Compartidos](#getters-compartidos) y [Acciones Compartidas](#acciones-compartidas).
```

Comment:
The anchors must be `[Getters Compartidos](#shared-getters) y [Acciones Compartidas](#shared-actions)`

Reply:
done